server/portal/settings/settings.py

```diff
@@ -749,3 +749,5 @@ def gettext(s): return s  # noqa:E731
 """
 if os.path.isfile(os.path.join(BASE_DIR, 'settings', 'settings_local.py')):
     from .settings_local import *  # noqa: F403, F401
+
+KEY_SERVICE_TOKEN = getattr(settings_secret, "_KEY_SERVICE_TOKEN", '')
```

Comment:
We had the key service creation actor handling this token previously - is it ok for us to be handling this here?

Reply:
I don't see why not-- ultimately the key service reactor can be called by the CEP admin client to push keys to arbitrary username/host combinations, so I don't really consider this an escalation of the portal's privilege.